dirty_equals/_datetime.py

```diff
@@ -43,14 +43,15 @@ def __init__(
         Examples of basic usage:
 
         ```py title="IsDatetime"
-        from datetime import datetime
+        from datetime import datetime, timezone
 
         from dirty_equals import IsDatetime
 
         y2k = datetime(2000, 1, 1)
         assert datetime(2000, 1, 1) == IsDatetime(approx=y2k)
-        # Note: this requires the system timezone to be UTC
-        assert 946684800.123 == IsDatetime(approx=y2k, unix_number=True)
+        assert 946684800.123 == IsDatetime(
+            approx=datetime(2000, 1, 1, tzinfo=timezone.utc), unix_number=True
+        )
```

Comment:
Some feedback on the devex with pytest-examples linting: It was annoying that I had to fix linting before I could see the actual test failure. I think that `test_docstrings` should run the actual test before linting.

Reply:
you can automate the updates uses `--update-examples`.

Reply:
Right, it took me a while to figure that out, and it's an extra obstacle in development, e.g. configuring my IDE. So I think reordering test_docstrings might still be slightly helpful. It could also be good for the Makefile to have an entry which runs tests with `--update-examples`, probably without coverage.

Reply:
Did the reordering, opened https://github.com/samuelcolvin/dirty-equals/issues/83 to discuss the rest, so I think this can be resolved now.